docs/Zendesk-batchsource.md

```diff
@@ -16,19 +16,18 @@ Configuration
 
 ### Basic
 
-**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata.
 
 **Admin Email:** Zendesk admin email.
 
 **API Token:** Zendesk API token. Can be obtained from the Zendesk Support Admin interface.
-Check out [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
-for API Token generation.
+For API Token generation, see the [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-).
 
 **Subdomains:** List of Zendesk Subdomains to read object from.
 
 **Object to Pull:** Objects to pull from Zendesk API.
 
-**Start Date:** Filter data to include only records which have Zendesk modified date is greater than
+**Start Date:** Filter data to include only records which have Zendesk modified date that is greater than
```

Comment:
Here's what's in the doc: Filter data to include only records that have a Zendesk modified date that is greater than or equal to the specified date.

Reply:
updated